src/ansible/roles/common/tasks/main.yml

```diff
@@ -12,9 +12,15 @@
     group: root
     mode: 0600
 
+- name: Create /data
+  file:
+    path: '/data'
+    state: directory
+    mode: 0700
+
 - name: Copy common data
   synchronize:
-    src: '{{ playbook_dir }}/../../data/'
+    src: '{{ role_path }}/../../../../data/'
```

Comment:
Path to data directory can also go to variable file.

Reply:
I had to touch this as I was running the playbook from other ansible and the plabook_dir was not where it expected. Having variable for data dir path would definitely make it cleaner.

Reply:
Done.